run.py

```diff
@@ -1044,6 +1044,36 @@ def supported_devices(
     def engine_manifest():
         return engine_manifest_loader.load_manifest()
 
+    @app.post(
+        "/validate_kana",
+        response_model=bool,
+        tags=["その他"],
+        summary="テキストがAquesTalk記法に従っているか判定する",
+        responses={
+            400: {
+                "description": "テキストが不正です",
+                "model": ParseKanaBadRequest,
+            }
+        },
+    )
+    def validate_kana(text: str):
+        """
+        テキストがAquesTalk記法に従っているかどうかを判定します.
```

Comment:
他のとこと名称などを合わせて「AquesTalkライクな記法」、と句点は「。」でお願いします！
あとエラーを返すことも伝えると丁寧かもです。こんな感じとか･･･？
```suggestion
        テキストがAquesTalkライクな記法に従っているかどうかを判定します。
        従っていない場合はエラーが返ります。
```